client/lib/billing-card/app/adapters/billing-card.js

```diff
@@ -0,0 +1,2 @@
+import TaskAdapter from 'tahi/adapters/task';
+export default TaskAdapter.extend();
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.